consensus/src/dag/order_rule.rs

```diff
@@ -42,7 +42,7 @@ impl OrderRule {
         dag_window_size_config: Round,
     ) -> Self {
         let commit_events = storage
-            .get_latest_k_committed_events(dag_window_size_config)
+            .get_latest_k_committed_events(10 * epoch_state.verifier.len() as u64)
```

Comment:
we probably should get this from some on-chain config

Reply:
Next PR.